build/common.targets

```diff
@@ -8,6 +8,10 @@
     <DefineConstants Condition="'$(TargetFrameworkIdentifier)' == '.NETFramework' Or ('$(TargetFrameworkIdentifier)' == '.NETCoreApp' and $([MSBuild]::VersionGreaterThanOrEquals($(TargetFrameworkVersion), 5.0)))">$(DefineConstants);IS_SIGNING_SUPPORTED</DefineConstants>
   </PropertyGroup>
 
+  <PropertyGroup Condition="'$(EnforceCodeStyleInBuild)' != '' And '$(IsOfficialBuild)' != 'true'">
+    <WarningsNotAsErrors>CS1591</WarningsNotAsErrors>
```

Comment:
This warning is for missing xml comments for type members.  
https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/compiler-messages/cs1591